Client/Frontend/Widgets/LoginTableViewCell.swift

```diff
@@ -322,3 +336,22 @@ extension LoginTableViewCell {
         iconImageView.image = UIImage(named: "faviconFox")
     }
 }
+
+extension LoginTableViewCell: UITextFieldDelegate {
+    func textFieldShouldReturn(textField: UITextField) -> Bool {
+        return self.delegate?.shouldReturnAfterEditingDescription(self) ?? false
```

Comment:
The docs for `textFieldShouldReturn` say
> `true` if the text field should implement its default behavior for the return button

so this should probably be `?? true` if there's no delegate set.